docs/reference/alert-schema.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[alert-schema]]
+
+== Alert schema
+
+{elastic-sec} stores alerts that have been generated by detection rules in hidden {es} indices.  In 7.x versions, the index pattern was `.siem-signals-<spacename>-*`.  In 8.x versions, the index pattern is `.alerts-security.alerts-<spacename>-*`, and the field names have changed. The following table includes the new names and provides a cross-reference to the previous field names.
+
+NOTE: Users are advised NOT to use the `_source field` in alert documents, but rather use the fields API to programmatically obtain the list of fields used in these documents. Learn more about {ref}/search-fields.html[retrieving selected fields from a search].
+
+|==============================================
+|7.x signal field |8.x alert field | Description
```

Comment:
Can we add in a column for field datatype?

Reply:
Hey @MikePaquette, what would go in that field datatype column? I need to be mindful of how many columns are in the table b/c due to some styling limitations beyond my control, the table will start to look skewed if it's too wide. It can be as long as needed, but 5 columns (currently 3 + the 2 you suggested) may cause some visual issues. The table already isn't wrapping the way I'd ideally like it to b/c of the long field names in the second column.

Reply:
I understand the challenges with adding more columns.  The field datatype is the Elasticsearch datatype used by the index mapping template for the alerts indices.  In other words, what kind of field is this (keyword, multi-field, long, date, boolean, etc.). If we can't add a column, perhaps we can add this in the description.  It would be necessary only for the non-ECS fields.  Note:  the ECS field reference includes this information in the text of the description like this:  
![image](https://user-images.githubusercontent.com/22619280/151970673-ae4de5fd-7285-4b0e-9ca4-218035f04c61.png)

Reply:
Added `Type` in the Description column.